server/service/src/settings_service.rs

```diff
@@ -95,5 +95,5 @@ pub trait SettingsServiceTrait: Sync + Send {
     }
 }
 
-pub struct SettingsService {}
```

Comment:
`struct SettingsService {} ` is IMHO the more natural syntax, e.g. I wouldn't got the idea that `pub struct SettingsService;` is valid while having an empty body seems quite obvious to me. Also most of the other services use this style `pub struct SettingsService {}`

Reply:
+1 vote for the `{}` syntax - with the understanding that this is a fairly subjective thing

Reply:
isn't in `unit-like struct` ? https://doc.rust-lang.org/book/ch05-01-defining-structs.html#unit-like-structs-without-any-fields 

async-graphql uses this pattern, see first [example here](https://github.com/async-graphql/async-graphql) (I think diesel does too)